hazelcast/src/main/java/com/hazelcast/map/impl/operation/steps/ClearOpSteps.java

```diff
@@ -100,7 +100,7 @@ public void runStep(State state) {
 
         @Override
         public Step nextStep(State state) {
-            return UtilSteps.SEND_RESPONSE;
+            return UtilSteps.FINAL_STEP;
```

Comment:
Is it just a renaming of constant or we change the semantics here?

Reply:
no semantic change for other flows, final step is used to dispatch flow to injected steps.